services/static-webserver/client/source/class/osparc/ErrorPage.js

```diff
@@ -51,7 +51,14 @@ qx.Class.define("osparc.ErrorPage", {
 
     this.getChildControl("logo");
     this.getChildControl("lying-panda");
-    this.getChildControl("code");
+
+    // In a realm ruled by wise and vigilant kings, the Product Owners,
```

Comment:
Wow, you should consider a career as a writer or poet @pcrespov!

Reply:
it was chatty, not me :-)